lib/vanagon/driver.rb

```diff
@@ -115,6 +117,49 @@ def prepare(workdir = nil)
       raise e
     end
 
+    def devkit_run
+          # Simple sanity check for the project
+      if @project.version.nil? or @project.version.empty?
+        raise Vanagon::Error, "Project requires a version set, all is lost."
+      end
+      @workdir = Dir.mktmpdir
+
+      if @devkit_target_specified.nil?
+        @engine.startup(@workdir)
+      end
+
+      puts "Target is #{@engine.target}"
```

Comment:
Honest question: wouldn't these prompts make more sense inside of each of these methods themselves, and all `devkit_run` does is enable them? That's a real good use of a mixin.